http/errors_test.go

```diff
@@ -102,6 +142,9 @@ func TestCheckError(t *testing.T) {
 			cmpopt := cmp.Transformer("error", func(e error) string {
 				if e, ok := e.(*influxdb.Error); ok {
 					out, _ := json.Marshal(e)
+					if e2, ok := e.Err.(http.RetriableError); ok {
```

Comment:
this is for the `cmp.Equal` that would otherwise panic: cannot handle unexported field at {*influxdb.Error}.Err.(*errors.errorString) ... I keep this unchanged in this PR